card/lib/generators/card/format/USAGE

```diff
@@ -0,0 +1,9 @@
+Description:
```

Comment:
This is just the original generators moving into a place where they show up in the help lists, etc.  Try out the generator help subcommands on the branch.